server/data/apps.json

```
@@ -1,4 +1,21 @@
 [
+  {
+    "appName": "Audicy",
+    "appType": [
+      "Discord Audio Streaming Bot"
+    ],
```

The app types have to match up with the app types listed in: https://github.com/Podcastindex-org/web-ui/blob/45cce47335def6226640f539e5ac3a2b45816a2d/server/data/apps-schema.json#L24-L45

```suggestion
    "appType": [
      "app",
      "podcast player"
    ],
```

I wonder if a `chat bot` type needs to be added to `apps-schema.json`